Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -542,6 +543,46 @@ private static void Clean(NugetPackageIdentifier package)
             }
         }
 
+        private static bool IsAlreadyImportedInEngine(NugetPackageIdentifier package) {
+            var alreadyImportedLibs = GetAlreadyImportedLibs();
+            var isAlreadyImported = alreadyImportedLibs.Contains(package.Id);
+            LogVerbose("Is package '{0}' already imported? {1}", package.Id, isAlreadyImported);
+            return isAlreadyImported;
+        }
+
+        private static HashSet<string> GetAlreadyImportedLibs() {
+            var lookupPaths = GetAllLookupPaths();
+            var libNames = lookupPaths
+                .SelectMany(directory => Directory.EnumerateFiles(directory, "*.dll", SearchOption.AllDirectories))
+                .Select(Path.GetFileName)
+                .Select(p => Path.ChangeExtension(p, null));
+            var result = new HashSet<string>(libNames);
+            LogVerbose("Already imported libs: {0}", string.Join(", ", result));
+            return result;
+        }
+
+        private static string[] GetAllLookupPaths() {
+            var executablePath = EditorApplication.applicationPath;
+            var roots = new[] {
+                // MacOS directory layout
+                Path.Combine(executablePath, "Contents"),
+                // Windows directory layout
+                Path.Combine(Directory.GetParent(executablePath).FullName, "Data")
+            };
+            var relativePaths = new[] {
```

Comment:
We should be able to select the right paths here based on the scripting backend settings, e.g. MonoBleedingEdge if we're targeting .Net 4.x and NetStandard if we're setup for .Net Standard.

Reply:
Oh, I see your comment on the related issue that MonoBleedingEdge was still being leveraged when targeting .Net Standard. Hmmm.